front/app/containers/Admin/projects/project/permissions/containers/ProjectVisibility/ProjectGroupsList.tsx

```diff
@@ -102,55 +106,53 @@ class ProjectGroupsList extends PureComponent<
   }
 
   componentDidMount() {
-    const { projectId } = this.props;
+    const { projectId, appConfiguration } = this.props;
     const locale$ = localeStream().observable;
-    const currentTenant$ = currentAppConfigurationStream().observable;
+
     const groups$ = getGroups().observable;
     const groupsProjects$ =
       projectGroupsByProjectIdStream(projectId).observable;
 
     this.subscriptions = [
-      combineLatest([
-        locale$,
-        currentTenant$,
-        groups$,
-        groupsProjects$,
-      ]).subscribe(([locale, currentTenant, groups, groupsProjects]) => {
-        const currentTenantLocales =
-          currentTenant.data.attributes.settings.core.locales;
-        const projectGroups = map(groupsProjects.data, (groupProject) => {
-          const group = find(
-            groups.data,
-            (group) => group.id === groupProject.relationships.group.data.id
-          ) as IGroupData;
-
-          return {
-            group_id: group.id,
-            group_project_id: groupProject.id,
-            title: getLocalized(
-              group.attributes.title_multiloc,
-              locale,
-              currentTenantLocales
-            ),
-            membership_count: group.attributes.memberships_count,
-          };
-        }).reverse();
-        const groupsOptions = this.getOptions(
-          groups,
-          groupsProjects,
-          locale,
-          currentTenantLocales
-        );
-        const loading = false;
-
-        this.setState({
-          locale,
-          currentTenantLocales,
-          projectGroups,
-          groupsOptions,
-          loading,
-        });
-      }),
+      combineLatest([locale$, groups$, groupsProjects$]).subscribe(
```

Comment:
Just to understand the approach:
Is it correct to say that you replaced all stream consumption of the tenant/appConfiguration stream except for those that are not within React components?

Reply:
Exactly!